fix for "undefined reference to 'Fixed_score_buffer<int> ...'"
cfr. https://github.com/bbuchfink/diamond/issues/260
--- diamond-0.9.24/src/dp/needleman_wunsch.cpp.orig	2019-06-04 09:17:04.256108175 +0200
+++ diamond-0.9.24/src/dp/needleman_wunsch.cpp	2019-06-04 09:17:15.046176190 +0200
@@ -171,6 +171,8 @@
 	return mtx.score_buffer();
 }
 
+template const Fixed_score_buffer<int>& needleman_wunsch(sequence query, sequence subject, int &max_score, const Local&, const int&);
+
 int needleman_wunsch(sequence query, sequence subject, int qbegin, int qend, int sbegin, int send, unsigned node, unsigned edge, Diag_graph &diags, bool log)
 {
 	const sequence q = query.subseq(qbegin, qend), s = subject.subseq(sbegin, send);
@@ -327,4 +329,4 @@
 	}
 	print_diag(i0, j0, l, score, diags, q, s);
 	print_hsp(hsp, TranslatedSequence(q));
-}
\ No newline at end of file
+}
